docs/src/explore/libraries/cli.md

```diff
@@ -14,6 +14,4 @@ The considered libraries:
 
 Probably the most widely used CLI library in Rust.
 
-With version 2.x, we'd probably want to use it with [Structops](https://github.com/TeXitoi/structopt) for deriving.
-
-But we can probably use 3.0, which is not yet stable, but is pretty close <https://github.com/clap-rs/clap/issues/1037>. This version comes with deriving attributes and also other new ways to build CLI commands.
+With version 2.x, we'd probably want to update clap to 3.x: this version comes with deriving attributes and also other new ways to build CLI commands (previously, deriving was only provided by [StructOpt](https://github.com/TeXitoi/structopt), which is now in maintenance mode).
```

Comment:
```suggestion
Comes with deriving attributes and also other new ways to build CLI commands (previously, deriving was only provided by [StructOpt](https://github.com/TeXitoi/structopt), which is now in maintenance mode).
```
let's just remove the mentions of the version as the stable v3 has been released a while back now